docs/application/dotnet/samples/machinelearning/overview.md

```diff
@@ -38,7 +38,18 @@ for TD:
 <table>
 	<tbody>
 		<tr>
-			<td>There is no sample.</td>
+			<td><img alt="" height="180" src="media/w83orientationdetection.png" width="180"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/MachineLearning/OrientationDetection" target="_blank"><strong>(W) OrientationDetection</strong></a></p>
+			<p>This sample application demonstrates how to use machine learning and sensor data using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.MachineLearning.Inference.Pipeline.html" target="_blank">Tizen.MachineLearning.Inference.Pipeline</a>.</p>
+			<p>It passes accelerometer sensor data stream to a neural network (tensorflow-lite model) and predicts one of four orientations of the device: <br>
+			<ol start="1">
+				<li>12 o'clock is upward.</li>
+				<li>3 o'clock is upward.</li>
+				<li>6 o'clock is upward.</li>
```

Comment:
Everything is upward? 6 o clock , 3, and 9?

Reply:
And why is the list as ordered list?

Reply:
This sample app shows a text one of "12 UP", "3 UP", "6 UP", or "9 UP", depending on the accelerometer sensor value.
Please check the demo here: https://github.com/nnstreamer/nnstreamer-example/tree/main/Tizen.NET/OrientationDetection

Unordered list might be more suitable?

Reply:
@annie-samsung Would you check the comments of anyi0527? I think.. it doesn't matter..

Reply:
@sgchoi5, @anyj0527, thanks for clarifying.
Can you change the list to unordered if there is no significance of it being listed in sequence?

Reply:
OK. will update it soon!